index.html

```diff
@@ -60,5 +60,172 @@ <h2>I’m Ahmed Glad to see you!</h2>
           
 
     </section>
+ 
+    <section class="works">
+      <div class="card-1">
+        <img src="assets/images/Snapshoot Portfolio-1.png" alt="Snapshoot-portfolio">
+        <div class="info-block">
+          <div class="client">
+            <h2 class="project-title-1">Tonic</h2>
+            <div class="details-1">
+              <div class="name"><p class="first-project-name">CANOPY</p></div>
+              <div class="curcle-1"></div>
+              <div class="role"><p class="first-project-role">Back End Dev</p></div>
+              <div class="curcle-2"></div>
+              <div class="year"><p class="first-project-year">2015</p></div>
+            </div>
+          </div>
+          <p class="summary-1">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <div class="languages-1">
+            <div class="html"><p class="first-project-html">html</p></div>
+            <div class="css"><p class="first-project-css">css</p></div>
+            <div class="js"><p class="first-project-js">javaScript</p></div>
+          </div>
+          <div class="action">
+           <button><p class="first-button">See Project</p></button>
+          </div>
+        </div>
+      </div>
+      <div class="card-2">
+        <img src="assets/images/Snapshoot Portfolio-2.png" alt="Snapshoot-portfolio">
+        <div class="info-block">
+          <div class="client">
+            <h2 class="project-title-2">Multi-Post
+              Stories</h2>
+            <div class="details-2">
+              <div class="name"><p class="second-project-name">CANOPY</p></div>
+              <div class="circle-1"></div>
+              <div class="role"><p class="second-project-role">Back End Dev</p></div>
+              <div class="curcle-2"></div>
+              <div class="year"><p class="second-project-year">2015</p></div>
+            </div>
```

Comment:
- Kindly make sure your design matches the Figma design by:
  - removing the space white space overflowing to the right :recycle: 
  - using the exact font family,size, weight, and colors used in the Figma(Poppins) :recycle: 
  - Using the right background color for the buttons (#fff)

![Screenshot from 2023-01-19 18-27-05](https://user-images.githubusercontent.com/97834160/213497837-4403b6bd-4a2b-49a4-9f1e-7995364791d1.png)